components/timezone/README.md

```diff
@@ -1,6 +1,98 @@
 # icu_timezone [![crates.io](https://img.shields.io/crates/v/icu_timezone)](https://crates.io/crates/icu_timezone)
 
-Types for resolving and manipulating timezones
+Types for resolving and manipulating time zones.
+
+## Fields
+
+In ICU4X, a formattable time zone consists of four different fields:
+
+1. The offset from GMT
+2. The time zone ID
+3. The meta zone ID
+4. The zone variant (standard or daylight time)
+
+### GMT Offset
+
+The GMT offset precisely states the time difference between the time zone in question and
+Greenwich Mean Time (GMT) or Coordinated Universal Time (UTC).
+
+In localized strings, it is often rendered as "GMT-6", meaning 6 hours less than GMT.
+
+### Time Zone
+
+The time zone ID corresponds to a time zone from the time zone database. The time zone ID
+usually corresponds to the largest city in the time zone.
+
+There are two mostly-interchangeable standards for time zone IDs:
+
+1. IANA time zone IDs, like `"America/Chicago"`
+2. BCP-47 time zone IDs, like `"uschi"`
+
+ICU4X uses BCP-47 time zone IDs for all of its APIs.
+
+### Meta Zone
```

Comment:
thought: still not convinced it should be two words cc @nordzilla 

(but let's not block merging this on that)